Packages/io.chainsafe.web3-unity/Editor/WebGLThreadPatcherInstaller.cs

```diff
@@ -36,7 +37,9 @@ static WebGLThreadPatcherInstaller()
     [MenuItem("ChainSafe SDK/Install WebGLThreadingPatcher", priority = 0)]
     public static void TryInstallThreadPatcher()
     {
-        Manifest manifest = JsonConvert.DeserializeObject<Manifest>(File.ReadAllText(ManifestPath));
```

Comment:
You would be better served wrapping this in a try/catch. Also, you should first check to ensure the ManifestPath exist before performing any operations.

Reply:
Manifest path is certain to exist since this is a Unity project, or else this script wouldn't run but I'll add the try catch...good catch xD

Reply:
changes made in latest commit